sites/all/themes/wille/templates/paragraphs/paragraphs-item--material-carousel--full.tpl.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Default theme implementation for a single paragraph item.
+ *
+ * Available variables:
+ * - $content: An array of content items. Use render($content) to print them
+ *   all, or print a subset such as render($content['field_example']). Use
+ *   hide($content['field_example']) to temporarily suppress the printing of a
+ *   given element.
+ * - $classes: String of classes that can be used to style contextually through
+ *   CSS. It can be manipulated through the variable $classes_array from
+ *   preprocess functions. By default the following classes are available, where
+ *   the parts enclosed by {} are replaced by the appropriate values:
+ *   - entity
+ *   - entity-paragraphs-item
+ *   - paragraphs-item-{bundle}
+ *
+ * Other variables:
+ * - $classes_array: Array of html class attribute values. It is flattened into
+ *   a string within the variable $classes.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_entity()
+ * @see template_process()
+ */
+
+  $bg_color = !empty($content['field_material_carousel_color'][0]['#markup']) ? $content['field_material_carousel_color'][0]['#markup'] : NULL;
```

Comment:
Eh, what's wrong with https://github.com/eReolen/ereolen.dk/blob/7a7d94037c72e6a55a30f509add662979decfd9f/sites/all/modules/breol/paragraphs/breol_carousel_paragraph/breol_carousel_paragraph.module#L11 ?

Eliminates the need for a template.

Reply:
How does it eliminate a need for a template if i need custom markup?
(    <div class="ding-carousel__background" style="background-color: <?php print $bg_color; ?>"></div>)